cypress/integration/create-post.test.ts

```diff
@@ -0,0 +1,45 @@
+import { HttpMethod } from "@lib/http-method";
+
+export {};
+
+describe("create post flow", () => {
+  beforeEach(() => {
+    cy.task("db:setup");
+  });
+
+  afterEach(() => {
+    cy.task("db:teardown");
+  });
```

Comment:
Is this required?

Reply:
I use it for the `/api/book/{algorithms}/` request, since the book needs to be in the database to proceed with searching. I'll look into mocking that portion out.

Reply:
I tried mocking it out, but then the POST request I make later doesn't work. I think that it would be better to just run the DB setup and teardown. This ensures that the frontend makes the correct requests to the API, and also requires less work, since the responses to the requests don't all have to be mocked out.